package.json

```diff
@@ -55,5 +55,30 @@
     "build": "npm install && grunt",
     "start": "grunt watch",
     "test": "grunt"
+  },
+  "commitplease": {
+    "components": [
+      "Docs",
+      "Tests",
+      "Build",
+      "Release",
```

Comment:
20ea8071b44c23e7915300eacc9ed958fbe4eaf5
de7702ac59889177b0dd622d3b5503004abf0258
1d931068d30b6a516b883cf7f420606de21eab68

They're distinct enough from Build to make this a worthwhile component, in my opinion.